ui/src/styles/_cards.scss

```diff
@@ -66,6 +66,27 @@
     border: var(--line);
   }
 
+  // dashboard edit menu cards and style
+  &--update {
```

Comment:
if this is specific to dashboard then maybe it makes sense to name it card--dashboard and/or add some logic where it can only be accessed within a dashboard layout (and make sure all pages in a dashboard layout have the same dashboard layout class)

Justy trying to stay away from specificity with global styles as much as possible